src/problems/MetalChem/metalcloud.cpp

```diff
@@ -0,0 +1,605 @@
+//==============================================================================
+// TwoMomentRad - a radiation transport library for patch-based AMR codes
+// Copyright 2020 Benjamin Wibking.
+// Released under the MIT license. See LICENSE file included in the GitHub repo.
+//==============================================================================
+/// \file popiii.cpp
+/// \brief Defines a test problem for Pop I/II star formation.
+/// Author: Piyush Sharda (Leiden University, 2024)
```

Comment:
You can use the Doxygen `\author` syntax here: https://www.doxygen.nl/manual/commands.html#cmdauthor